test/unit/component/parser/json/BpmnJsonParser.label.font.test.ts

```diff
@@ -64,6 +64,7 @@ describe('parse bpmn as json for label font', () => {
                 Bounds: { x: 362, y: 232, width: 36, height: 45 },
                 BPMNLabel: {
                   id: 'label_id',
+                  isExpanded: true,
```

Comment:
:question: same remarks as for `BpmnJsonParser.label.bounds.test.ts`

Reply:
I added some scpecific tests for call Activity, and after remove them in order to come back to the previous tests,. Maybe I forgot some adding 😉